### PR TITLE
(fix|COS-3818): Fix condition disabling row selection while editing cell

### PR DESCRIPTION
**Pull request by Kasia9090**
What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

## Summary by CodeRabbit

- **Bug Fixes**
  - Adjusted the logic for enabling keyboard shortcuts in the FinderTable component. Keyboard shortcuts will now only be enabled if both editing and filtering modes are disabled.

### CODE REVIEW

File: packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx [hunk 1]
```diff
@@ -194,7 +194,7 @@ export const FinderTable = observer(() => {
       onFocusedRowChange={setFocusIndex}
       onSelectedIndexChange={setSelectedIndex}
       onSelectionChange={handleSelectionChange}
-      enableKeyboardShortcuts={!isEditing || !isFiltering}
```
Comment (acalinica) [inline packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx:197]: this is not right. I also want to block keyboard events if I'm only searching. The correct fix would be to do store.ui.setIsEdittingFilterTable(true) when you focus the editable input (whicever might be) and set it bag to false when you blur it.

**Review by acalinica — CHANGES_REQUESTED** [review]

File: packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx (hunk 1 above)
Comment (Kasia9090) [reply packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx:197]: that is already done and the condition is correct
if I'm editing then is edit is true so !isEdit is false so I go to or condition and check !isFiltering and cause I'm not filtering at this moment isFiltering is false so !isFiltering is true -> that results in events not being blocked

Comment (Kasia9090) [reply packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx:197]: in other words I want keyboard shortcuts to be enabled only when user is not editing and not searching in given moment

Comment (acalinica) [reply packages/apps/frontera/src/routes/organizations/src/components/FinderTable/FinderTable.tsx:197]: ok

**Review by acalinica — APPROVED** [review]

**Pull request merged**